src/App.tsx

```diff
@@ -1,61 +1,34 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { NewTodo } from './components/NewTodo/NewTodo';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  function addNewTodo(title: string, userId: number, completed = false) {
+    const maxId = Math.max(...todos.map(item => item.id));
+
+    const newT: Todo = {
+      id: maxId + 1,
+      title: title,
+      completed: completed,
+      userId: userId,
+    };
+
+    setTodos(current => [...current, newT]);
+  }
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. Variable 'title' and 'userId' are too generic and could be more descriptive. Also, the 'completed' parameter can be named more explicitly to indicate it's a default value for the new todo's completion status.